nengo_gui/components/value.py

```diff
@@ -22,8 +23,20 @@ def __init__(self, obj):
         # the pending data to be sent to the client
         self.data = []
 
+        # grab the output of the object
+        self.output = self.obj
+        if isinstance(self.obj, spa.module.Module):
+            self.output = self.obj.outputs['default'][0]
+        elif isinstance(self.obj, nengo.network.Network):
+            if self.obj.output != None:
```

Comment:
Won't this crash if ```self.obj``` is a Network that doesn't have an attribute ```output```?  Did you mean ```if hasattr(self.obj, 'output'):```?